cmd/go-vcs/go-vcs.go

```diff
@@ -309,23 +310,38 @@ func main() {
 		}
 
 	case "branches":
-		if len(args) != 0 {
-			log.Fatal("branches takes no arguments.")
+		if len(args) > 1 {
+			log.Fatal("branches takes 0 or 1 arguments.")
+		}
+
+		// Open using go/vcs to figure out VCS type (git, hg).
+		r := vcs2.New(".")
```

Comment:
Yeah, extracting the package into its own repo would be great. Not a big deal either way, since this is in the command, not the lib (and Godeps users of this lib wouldn't pull in the cmd deps).